react/modules/graphiql.md

```diff
@@ -66,69 +69,70 @@ You are free to choose your own way of developing the project and the repository
 
 - the application quality is characterized by the elaboration of details, attention to typography (no more than three fonts per page, font size of at least 14 px, optimal [font and background contrast](https://snook.ca/technical/colour_contrast/colour.html)), carefully selected content
 - adaptive layout, the minimum page width of the application should be 320px
 - interactivity of elements users can interact with; element hover effects; usage of different styles for the active and inactive state of the element; smooth animations
-- the unity of styles of all pages of the application - the same fonts, button styles, indents, the same elements on all pages of the application have the same appearance and layout. Item colors and background images may vary. In this case, colors should be from the same palette, and background images from the same collection.
+- the unity of styles of all pages of the application - the same fonts, button styles, indents, and the same elements on all pages of the application have the same appearance and layout. Item colors and background images may vary. In this case, colors should be from the same palette, and background images from the same collection.
 
 ## Description of function blocks
 
 ### General requirements
 
-- Errors from the api side (Not found, unhandled rejection, etc) should be displayed in a user-friendly format (toast, pop-up, or something like this - up to your decision).
-- Localization (at least 2 languages). You should be able to change the language by clicking on the toggler/select in header. Localization should be implemented using Context API. No 3rd party / open source solutions are allowed.
+- Errors from the API side (Not found, unhandled rejection, etc) should be displayed in a user-friendly format (toast, pop-up, or something like that - up to your decision).
+- Localization (at least 2 languages). You should be able to change the language by clicking on the toggler/select in the header. Localization should be implemented using Context API. No 3rd party / open-source solutions are allowed.
 
 ### Welcome page
 
-- If user is not authorized, the page should contain a link to Sign In / Sign Up page
-- If user is authorized, the page should contain a link to the Main page
+- If the user is not authorized, the page should contain a link to Sign In / Sign Up page.
+- If the user is authorized, the page should contain a link to the Main page.
 
 ### Header
 
-- All Private routes should contain sticky header. Moment when it become sticky (if there is a scroll on a page) should be animated: color can be changed or it's height can become smaller. [Animated sticky header](https://www.youtube.com/watch?v=hR8UW5CvYgw)
+- All Private routes should be present in a sticky header. The moment it becomes sticky (if there is a scroll on a page) should be animated: color can be changed or its height can become smaller. [Animated sticky header](https://www.youtube.com/watch?v=hR8UW5CvYgw)
```

Comment:
Was the meaning of this sentence changed intentionally? The original meaning was that private routes (in our case only the editor page) should have a sticky header (which I don't think makes much sense).
And you changed it so that a sticky header should be on the welcome page, right?

Reply:
Original sentence made no sense to me, but I guess this is not better either.

Reply:
Made changes after discussion in chat.